xnmt/input_reader.py

```diff
@@ -190,11 +190,12 @@ class H5Reader(InputReader, Serializable):
   * sents[sent_id][word_ind,feat_ind] if transpose=True
 
   Args:
-    transpose (bool):
-    feat_from (int):
-    feat_to (int):
-    feat_skip (int):
-    word_skip (int):
+    transpose (bool): whether inputs are transposed or not.
+    feat_from (int): use feature dimensions in a range, starting at this index (inclusive)
+    feat_to (int): use feature dimensions in a range, ending at this index (exclusive)
+    feat_skip (int): stride over features
+    word_skip (int): stride over words
```

Comment:
Are we assuming that each column in the feature vector corresponds to a "word" with this notation? Maybe "time", "timestep", or something else would be better?

Reply:
Sure, thanks for the hint!